app/uk/gov/hmrc/plasticpackagingtax/registration/controllers/group/GroupMemberGrsController.scala

```diff
@@ -78,7 +77,7 @@ class GroupMemberGrsController @Inject() (
                   case Left(error) => throw error
                 }
               case DUPLICATE_SUBSCRIPTION =>
-                Future(Redirect(pptRoutes.NotableErrorController.duplicateRegistration()))
+                Future(Redirect(groupRoutes.NotableErrorController.groupMemberAlreadyRegistered()))
```

Comment:
I think you will need to create a "GroupError" object here are save it before redirecting - see comment on the new view.